### DISCUSSION TITLE
Any way to hide previous/next months days?

### DISCUSSION THREAD

**Discussion by Bretthansenpc** [totaldebug/atomic-calendar-revive#962] [Support] [answered]
I'm trying to do a fridge calendar for the family and I'm curious if there is a way to hide the numbers of the previous end of the month, and start of the next? 

The family is used to "at a glance" calendars haha

**Comment by marksie1988** [comment, marked as answer]
I think if i understand correctly you will want this option `showLastCalendarWeek` in calendar mode
